### DISCUSSION TITLE
Flet  in Rust

### DISCUSSION THREAD

**Discussion by gitmatsika** [flet-dev/flet#2717] [Q&A]
Do you have plans to have flet for rust in future just like python

**Comment by baseplate-admin** [comment]
Hi, I think current priority is to get everything working in `python`.

Please note that porting a framework to another language takes an enormous amount of effort.

<sub>maybe you can help port `flet` to `rust`? </sub>

**Reply by omolayoabegunde** [reply]
Well done sir

**Comment by gitmatsika** [comment]
Noted . Thnk you so much for the work
